README.md

```diff
@@ -76,52 +51,45 @@ You can disable HLint and also control the maximum number of reported problems,
 If the liquid haskell executable is installed, enable using it to
 process haskell files on save.
 
-```
+```json
 "languageServerHaskell.liquidOn": true,
 ```
 
-#### HIE Wrapper
-
-Furthermore, the extension supports multiple ways of initializing hie, depending on your needs. 
-The default one uses the `hie-vscode.[sh|bat]` file to start hie through. 
-This will attempt to start via `hie-wrapper` if it is on the vscode path, otherwise fall back to `hie`.
-The`hie-wrapper` executable is installed using `stack ./install.hs hie` or `./cabal-hie-install hie`, 
-together with the other GHC version specific `hie-8.x` executables.
-This will take precedence over `hieExecutablePath`.
-
-#### Custom Wrapper
-
-If you need more control, and want to have a custom wrapper, either in your specific project or somewhere else on your computer, you can set a custom wrapper via,
-
-```json
-"languageServerHaskell.useCustomHieWrapper": true,
-"languageServerHaskell.useCustomHieWrapperPath": "~/wrapper-in-home.sh",
-```
+### HIE Variant
 
-There are a few placeholders which will be expanded:
+Since `0.40` the extension no longer uses the wrapper. It has been replaced by a selection over
```

Comment:
Wait what? It doesnt? Is always `hie`/`ghcide`/`hls` selected now or the respective wrapper?

Reply:
Mmm, the wrapper was removed, the selector is used to know what executable should be called
In fact the wrapper could be replaced with the hie executable field, no?

Reply:
Ok, just wanted to make sure. I probably dont know what the purpose of the wrapper was then? 
To me it sounds like `hie-wrapper` and `haskell-language-server-wrapper` are obsolete now.

Reply:
Mmm but we need to switch between hie and hls versions to get the appropiate one for the project ghc, no?

Reply:
I think so, too. We need the `-wrapper` executables, right?  (I dont use them at the moment)
This documentation sentence is a bit confusing to me.

Reply:
Oh, i see, the wrappers executables are still needed although the fields to set the custom wrapper are not, i will rephrase

Reply:
@fendor i've removed the reference to wrapper entirely cause, to be precise, the new field does not replace the custom wrapper

Reply:
Nice! Thanks!